adoc/CaaSP_DH2X_Install_Guide.adoc

```diff
@@ -84,160 +72,193 @@ The following software is needed
-
++
 image::028-SCT-CaaSP.png[itle="Select Nodes and Roles 3",480,640,scaledwidth=80%,align=center]
 
-Bootstrap the Kubernetes cluster.
-
+. Bootstrap the Kubernetes cluster.
++
 image::030-SCT-CaaSP.png[title="Confirm Bootstrap",480,640,scaledwidth=80%,align=center]
 
-After a successful bootstrap, the screen below is shown.
-
+. After a successful bootstrap, the screen below is shown.
++
 image::032-SCT-CaaSP.png[title="Cluster Status Nodes",480,640,scaledwidth=80%,scaledwidth=80%,align=center]
 
 
+==== Installing the Jump Host
 
-==== Installation of the Jump Host
+For further configuration and deployments on SUSE CaaS Platform it is highly recommended to install a jump host, also called a __management host__.
+This section describes installing the necessary tools to be able to deploy SAP Data Hub 2 on SUSE CaaS Platform.
 
-For further configuration and deployments on SUSE CaaS Platform it is highly recommended to install a jump or management host.
-This chapter describes the installation of the necessary tools to be able to deploy SAP Data Hub 2 on the SUSE CaaS Platform environment.
+. Install SUSE Linux Enterprise Server 12 SP4+ or SUSE Linux Enterprise Server 15 as the operating system on the jump host.
 
-Install SUSE Linux Enterprise Server 12 SP4+ or SUSE Linux Enterprise Server 15 as operating system on the jump host.
-Register your installation against the SUSE Customer Center (SCC) or your local SMT or Repository Mirroring Tool (RMT).
-Register the Container Module included in the SUSE Linux Enterprise Server subscription.
+. Register your installation against the SUSE Customer Center (SCC) or your local SMT (Subscription Management Tool) or RMT (Repository Mirroring Tool).
 
+. Register the Container Module included in the SUSE Linux Enterprise Server subscription:
++
 ----
 # SUSEConnect -p x86_64/SLE-Container-Module
 ----
 
-Install Docker from the Container Module:
-
+. Install Docker from the Container Module:
++
 ----
 # zypper in docker
 ----
 
-Download *kubectl* matching the Kubernetes version of your SUSE CaaS Platform installation.
+. Download *kubectl* matching the Kubernetes version of your SUSE CaaS Platform installation.
 Download *kubectl 1.10.11* or higher.
-
++
 ----
 $ curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.10.11/bin/linux/amd64/kubectl
 $ sudo mv kubectl /usr/bin/kubectl
 $ sudo chmod a+x /usr/bin/kubectl
 ----
 
-Download *kubectl-configuration* from the Velum-Dashboard.
 
+. Download the `kubeconfig` file from the Velum Dashboard.
++
 image::032a-SCT-CaaSP.png[title="kubeconfig 1",480,640,scaledwidth=80%,align=center]
-
++
 image::033-SCT-CaaSP.png[title="kubeconfig 2",480,640,scaledwidth=80%,align=center]
-
++
 image::034-SCT-CaaSP.png[title="kubeconfig 3",480,640,scaledwidth=80%,align=center]
 
+// ^FIXME: Is the order correct? 1. Status page->2. Login page->3. File download
+// ^FIXME 2: Is there really a necessity for three _uncropped_ screenshots?
```

Comment:
Fixmes for Ulrich...

Reply:
@FIXME 1
The order is correct: velum ui (1) -> login of dex ui (2)-> file download (3)
That's the way how CaaSP 3 worked. ;)
@FIXME 2:
If there is a way to do with cropped images please do so. Thx.

Reply:
Done.